scripts/users.sh

```diff
@@ -10,3 +10,6 @@ echo %misp ALL=NOPASSWD:ALL > /etc/sudoers.d/misp
 ##chmod 0440 /etc/sudoers.d/thehive
 chmod 0440 /etc/sudoers.d/misp
 
+echo "--- Creating zmqs user"
+useradd -U -G www-data -m -s /bin/bash  zmqs
+echo zmqs:zmqs"Password1234" | chpasswd
```

Comment:
Does it require a new user with password?
This is potentially scary ;)

Reply:
Well yes, I should definitely remove that line :)